docs/reference/data-streams/tsds-reindex.asciidoc

```diff
@@ -0,0 +1,292 @@
+copied, there is limited risk of data loss due to lack of replicas.
+
+Using the example above as source TSDS, the template for the destination TSDS would be:
+
+[source,console]
+----
+POST /_component_template/destination_template
+{
+  "template": {
+    "settings": {
+      "index": {
+        "number_of_replicas": 0,
+        "number_of_shards": 4,
+        "mode": "time_series",
+        "routing_path": [ "metricset" ],
+        "time_series": {
+          "end_time": "2023-09-01T14:00:00.000Z",
+          "start_time": "2023-09-01T06:00:00.000Z"
+        }
+      }
+    },
+    "mappings": {
+      "properties": {
+        "@timestamp": { "type": "date" },
+        "metricset": {
+          "type": "keyword",
+          "time_series_dimension": true
+        },
+        "k8s": {
+          "properties": {
+            "tx": { "type": "long" },
+            "rx": { "type": "long" }
+          }
+        }
+      }
+    }
+  }
+}
+
+POST /_index_template/2
+{
+  "index_patterns": [
+    "k8s*"
+  ],
+  "composed_of": [
+    "destination_template"
+  ],
+  "data_stream": {}
+}
+----
+// TEST[continued]
+
+[discrete]
+[[tsds-reindex-op]]
+==== Reindex
+
+Invoke the reindex api, for instance:
+
+[source,console]
+----
+POST /_reindex
+{
+  "source": {
+    "index": "k8s"
+  },
+  "dest": {
+    "index": "k9s",
+    "op_type": "create"
+  }
+}
+----
+// TEST[continued]
+
+[discrete]
+[[tsds-reindex-restore]]
+==== Restore the destination index template
+
+Once the reindexing operation completes, restore the index template for the destination TSDS as follows:
+
+* Remove the overrides for `index.time_series.start_time` and `index.time_series.end_time`.
+* Restore the values of `index.number_of_shards index`, `index.number_of_replicas`  and  `index.lifecycle.name` as
+applicable.
+
+Using the previous example, the destination template is modified as follows:
+
+[source,console]
+----
+POST /_component_template/destination_template
+{
+  "template": {
+    "settings": {
+      "index": {
+        "number_of_replicas": 2,
+        "number_of_shards": 2,
+        "mode": "time_series",
+        "routing_path": [ "metricset" ]
+      }
+    },
+    "mappings": {
+      "properties": {
+        "@timestamp": { "type": "date" },
+        "metricset": {
+          "type": "keyword",
+          "time_series_dimension": true
+        },
+        "k8s": {
+          "properties": {
+            "tx": { "type": "long" },
+            "rx": { "type": "long" }
+          }
+        }
+      }
+    }
+  }
+}
+----
+// TEST[continued]
+
+Next, Invoke the `rollover` api on the destination data stream without any conditions set.
+
+[source,console]
+----
+POST /k9s/_rollover/
+----
+// TEST[continued]
+
+This creates a new backing index with the updated index settings. The destination data stream is now ready to accept new docs.
+
+Note that the initial backing index can still accept docs within the range of timestamps derived from the source data
```

Comment:
Maybe also add a note about data streams containing downsampled indices?

Reply:
I added one at the top, you want another one here? No objection, just checking..

Reply:
oops missed that... never mind